Gemfile

```diff
@@ -51,6 +51,11 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+end
+
+group :test do
+  gem 'rspec-rails'
 end
 
```

Comment:
- [ ] `README.md` file: To make your documentation be more professional, I think it would be better if you update some content information of sub-sections of the `## Getting Started` section with appropriate and correct information as some of them miss needed information and others have wrong information.
  - I suggest you complete `### Usage` sub section with the command of running your server locally i.e. `rails server` or `rails s`.
![Screenshot from 2023-07-27 08-22-15](https://github.com/BranBayou/blog-app/assets/83042742/29cf02b0-bc73-4611-819d-c83c94e1de3a)

  - I suggest you remove this command ``rails server`` from the `### Run tests` sub-section it is supposed to run the server and not running tests.
![Screenshot from 2023-07-27 08-22-32](https://github.com/BranBayou/blog-app/assets/83042742/d1730cec-c5ad-4e75-808e-e2a0286edfba)

  - An for other sections where you don't have information to fill in, I suggest you remove them/it from now and add them later if you find information to add there related to your project(e.g.: `Deployment`,... )
![Screenshot from 2023-07-27 08-22-42](https://github.com/BranBayou/blog-app/assets/83042742/f41c3f66-a5da-4840-a1ee-37a8a2c6b122)